packages/core/src/components/button/_button.scss

```diff
@@ -191,7 +191,7 @@ Styleguide pt-button
     #{$icon-classes} {
       margin-right: $button-icon-spacing-large;
 
-      &.pt-align-right {
+      &:last-child {
```

Comment:
skeptical of this change... using the `.pt-align-right` to change margins is much more reliable (though slightly more work for users) than affecting the last-child without thinking.

Reply:
@giladgray ok, so we should bias towards being more verbose in the html, which would lead to inadvertently selecting the wrong element?